src/bio/bio_xstream.c

```diff
@@ -1632,6 +1638,9 @@ scan_bio_bdevs(struct bio_xs_context *ctxt, uint64_t now)
 
 		D_INFO("Detected hot plugged device %s\n",
 		       spdk_bdev_get_name(bdev));
+		/* Print a console message */
+		D_PRINT("Detected hot plugged device %s\n",
+			spdk_bdev_get_name(bdev));
```

Comment:
I like the console print, but the device name might not be needed. It prints something like the following:
"daos_io_server:0 Detected hot plugged device HotInNvme0n1"
which is not very helpful? Can we print UUID here instead by chance, or maybe no device name just "Detected device hot plug event" ?

Reply:
The device name is helpful for developer to tell which ctrlr/ns is hotplugged. At this point, we only detected a new SPDK dev, don't know device ID yet. The device ID will be loaded from blobstore later, and the debug log will have device ID information once device ID is loaded. (but I didn't print more console messages)